src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Models/KeyValueSelector.cs

```diff
@@ -24,6 +28,11 @@ public class KeyValueSelector
         /// </summary>
         public string SnapshotName { get; set; }
 
+        /// <summary>
+        /// A filter that determines what tags to require when selecting key-values for the the configuration provider.
+        /// </summary>
+        public IEnumerable<string> TagsFilters { get; set; }
```

Comment:
```suggestion
        public IEnumerable<string> TagsFilter { get; set; }
```

JS sdk uses the name `tagsFilter`. [ref](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/appconfiguration/app-configuration/src/models.ts#L349)

Reply:
Interesting. I planned to use this name for JS provider.